src/Packetery/Module/HookHandler.php

```diff
@@ -76,6 +86,12 @@ public function updateOrder( $wcOrderId ): void {
 			$this->orderRepository->delete( (int) $wcOrderId );
```

Comment:
Udělejme to nakonec tak, že pokud je objenávka smazána, metoda se ukončí. Je zbytečné pokračovat.

Reply:
Hotovo